vulnerabilities/tests/test_postgres_workaround.py

```diff
@@ -0,0 +1,432 @@
+from datetime import datetime
+from datetime import timezone
+
+import pytest
+import pytz
+from packageurl import PackageURL
+from univers.version_constraint import VersionConstraint
+from univers.version_range import MavenVersionRange
+from univers.versions import MavenVersion
+
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
+from vulnerabilities.importer import Reference
+from vulnerabilities.importer import VulnerabilitySeverity
+from vulnerabilities.models import Advisory
+from vulnerabilities.severity_systems import ScoringSystem
+
+data = AdvisoryData(
+    aliases=["CVE-2020-8908", "GHSA-5mg8-w23w-74h3"],
+    summary="lore ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur excepteur sint occaecat cupidatat non proident sunt in culpa qui officia deserunt mollit anim id est laborum",
```

Comment:
Please either wrap this long line ...
```suggestion
    summary="""lore ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut labore et
             dolore magna aliqua ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
             ea commodo consequat duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu 
             fugiat nulla pariatur excepteur sint occaecat cupidatat non proident sunt in culpa qui officia deserunt 
             mollit anim id est laborum""",
```

or make it very long this way:

```suggestion
    summary="lore ipsum dolor sit amet consectetur adipiscing elit sed do" * 10000,
```


the point is that we want something long enough that used to fail before... Did you check these value are failing without the patch?

Reply:
Added the failing test PR(before this patch)